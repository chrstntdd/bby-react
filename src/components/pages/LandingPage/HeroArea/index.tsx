import * as React from 'react';
import { Link } from 'react-router-dom';

import './hero-area.scss';

const HeroArea = () => (
  <section id="hero-atf">
    <div className="button-container">
      <Link id="sign-up-button" to="/sign-up">
        <button id="btn-signup">Sign Up</button>
      </Link>

      <Link id="sign-in-button" to="/sign-in">
        <button id="btn-signin">Sign In</button>
      </Link>
    </div>
    <div className="hero-img" />
    <div className="hero-content">
      <h1>Quantified</h1>
      <h2>A streamlined system for efficiently and accurately managing retail inventory.</h2>
    </div>
  </section>
);

export default HeroArea;
