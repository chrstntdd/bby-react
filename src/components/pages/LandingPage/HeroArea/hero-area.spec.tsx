import * as React from 'react';
import { shallow } from 'enzyme';
import { MemoryRouter as Router } from 'react-router-dom';
import createRouterContext from 'react-router-test-context';

import HeroArea from './';

describe('<HeroArea/>', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<HeroArea />);

    expect(wrapper).toMatchSnapshot();
  });
  it('should contain the hero image div', () => {
    const wrapper = shallow(<HeroArea />);
    const heroImageDiv = wrapper.find('.hero-img');
    expect(heroImageDiv).not.toBe(null);
  });
});
