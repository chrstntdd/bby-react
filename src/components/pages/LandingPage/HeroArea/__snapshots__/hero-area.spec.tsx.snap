// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeroArea/> should render without crashing 1`] = `
<section
  id="hero-atf"
>
  <div
    className="button-container"
  >
    <Link
      id="sign-up-button"
      replace={false}
      to="/sign-up"
    >
      <button
        id="btn-signup"
      >
        Sign Up
      </button>
    </Link>
    <Link
      id="sign-in-button"
      replace={false}
      to="/sign-in"
    >
      <button
        id="btn-signin"
      >
        Sign In
      </button>
    </Link>
  </div>
  <div
    className="hero-img"
  />
  <div
    className="hero-content"
  >
    <h1>
      Quantified
    </h1>
    <h2>
      A streamlined system for efficiently and accurately managing retail inventory.
    </h2>
  </div>
</section>
`;
